awx/ui_next/src/screens/Template/JobTemplateDetail/JobTemplateDetail.jsx

```diff
@@ -156,6 +157,28 @@ class JobTemplateDetail extends Component {
       </TextList>
     );
 
+    const renderMissingDataDetail = value => (
```

Comment:
I think instead of re-using the `Detail` component and passing it a prop you can just create a `MissingDetail` styled component by extending it:

const MissingDetail = styled(Detail)`
  color: #c9190b;
`;

And then using it in the `render()` method.